### ISSUE TITLE
Unable to cy.get a form bootstrap custom switch

### ISSUE THREAD

**Issue by AoNoOokami** [cypress-io/cypress#9619]
### Current behavior

When testing an Angular form using Bootstrap 4 custom switches, it is impossible to interact with those elements. The selector playground doesn't seem to find them either. It sees the label or the div containing the input.

### Desired behavior

The toggle switch should be selectable with `cy.get` instruction.

### Test code to reproduce

Using a basic Cypress project, you can use this url to reproduce the behaviour: https://getbootstrap.com/docs/4.2/components/forms/#switches and try to toggle the example switch.

Template:
```html
<div class="custom-control custom-switch">
  <input type="checkbox" class="custom-control-input" id="customSwitch1">
  <label class="custom-control-label" for="customSwitch1">Toggle this switch element</label>
</div>
<div class="custom-control custom-switch">
  <input type="checkbox" class="custom-control-input" disabled id="customSwitch2">
  <label class="custom-control-label" for="customSwitch2">Disabled switch element</label>
</div>
```

Instruction that should work:
```js
cy.get('#customSwitch1').click()
```

```js
cy.get('#customSwitch1').check()
```

```js
cy.get('#customSwitch1').uncheck()
```

### Versions

v4.12.1

**Comment by bahmutov** [comment]
Can you try Cypress v6.1.0 which is the latest version please?

**Comment by AoNoOokami** [comment]
Hello, I installed the version 6.1.0 but the behaviour remains the same.

**Comment by jennifer-shehane** [comment]
In the given example, the checkbox *is hidden*, so you see an error in Cypress about the element being covered and not directly interactable. 

<img width="1222" alt="Screen Shot 2020-12-14 at 4 13 30 PM" src="https://user-images.githubusercontent.com/1271364/102065942-9175df80-3e27-11eb-9613-c38d923832a7.png">

First, the error notifies you that it is covered by the fixed header element. Since Cypress scrolls the element to the top, the checkbox is under the header, we can change the scrollBehavior to only scroll to the center. 

```js
cy.get('#customSwitch1').check({ scrollBehavior: 'center' })
```

Now the error says that the checkbox is covered by the label element. And it in fact is. 

<img width="1270" alt="Screen Shot 2020-12-14 at 4 18 20 PM" src="https://user-images.githubusercontent.com/1271364/102066288-047f5600-3e28-11eb-8ff1-78f6e7369298.png">

So, I'd look at how this switch is implemented in the first place, it seems that the event to check the checkbox is triggered after clicking the parent .custom-switch element, so let's click that. 

```js
it('test', () => {
  cy.visit('https://getbootstrap.com/docs/4.2/components/forms/#switches')
  cy.get('#customSwitch1').parent('.custom-switch').click()
})
```

This is clicking in the center of the element (the default), which means it's not hitting the hitbox that the custom switch is looking for, so I would try clicking on the very left of the element by passing `position: 'left'`.

```js
it('test', () => {
  cy.visit('https://getbootstrap.com/docs/4.2/components/forms/#switches')
  cy.get('#customSwitch1').parent('.custom-switch').click({ position: 'left' })
  cy.get('#customSwitch1').should('be.checked')  // ✅ works
})
```

<img width="1132" alt="Screen Shot 2020-12-14 at 4 24 52 PM" src="https://user-images.githubusercontent.com/1271364/102067008-ed8d3380-3e28-11eb-82aa-498c2137ed5f.png">

Or you could just force the click like below and that works just the same.

```js
it('test', () => {
  cy.visit('https://getbootstrap.com/docs/4.2/components/forms/#switches')
  cy.get('#customSwitch1').check({ force: true })
  cy.get('#customSwitch1').should('be.checked')  // ✅ works
})
```

Basically, you will need some understanding on how the component works in order to test it.

**Comment by AoNoOokami** [comment]
Thanks for your quick and kind response.
